lib/3scale_toolbox/commands/import_command/openapi.rb

```diff
@@ -30,6 +30,7 @@ def self.command
               option  :t, 'target_system_name', 'Target system name', argument: :required
               flag    nil, 'activedocs-hidden', 'Create ActiveDocs in hidden state'
               flag    nil, 'skip-openapi-validation', 'Skip OpenAPI schema validation'
+              flag    nil, 'prefix-matching', 'Use prefix matching instead of full path matching on mapping rules derived from openapi operations'
```

Comment:
instead of "full path matching", "strict matching" naming is widely used

Reply:
:+1: